app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
```

Comment:
Code Style: According to the checklist, you should not repeat abstract methods from parent if you have an abstract class that inherits another abstract class. Since `Player` class is an abstract class, you should not inherit `ABC` again in `Dwarf` class. You can simply write `class Dwarf(Player):`.